docs/tasks/File.md

```diff
@@ -39,14 +39,19 @@ $this->taskReplaceInFile('config.yml')
  ->regex('~^service:~')
  ->to('services:')
  ->run();
+
+ $this->taskReplaceInFile('box/robo.txt')
```

Comment:
please move this example to dockblock of the class.
this file is generated from it